<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterWindow</class>
 <widget class="QMainWindow" name="ParameterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>808</width>
      <height>608</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>700</y>
      <width>192</width>
      <height>28</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QPushButton" name="updateImageButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>660</y>
      <width>192</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Image</string>
    </property>
   </widget>
   <widget class="QSlider" name="exposureSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>700</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure (ms)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ROIButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Region of Interest</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>665</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(Press Enter or Space to confirm, press C to cancel)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
